distributed/worker.py

```diff
@@ -708,8 +708,14 @@ def _register_with_scheduler(self):
         logger.info("-" * 49)
         while True:
             if self.death_timeout and time() > start + self.death_timeout:
+                logger.exception(
```

Comment:
I can't easily comment on it here, but down in https://github.com/dask/distributed/pull/2784/files#diff-3f78e3c9a9a81b6355c8e594dce0b8f3R766 we also may handle TimeoutError. In that case we `logger.info` and move on.

I'm not quite sure when that is reached, but I think since we're under this `while True` we'll still end up here.